gamelogs-client/src/components/Navbar.tsx

```diff
@@ -9,7 +9,7 @@ const Navbar = () => {
                 <div className="flex items-center">
                 <a href="/login">Log In</a>
                 </div>
-                <form action="/login">
+                <form action="/profile">
```

Comment:
Is this what you mean by it redirecting it to the profile url? If this is not intended, it can just simply be switched back to `/login`. One thing you might want to consider also, is checking if the user is logged in already (lmk if you want to know how this is done). If the user isn't logged in, show the sign up and login button (they effectively do the same thing, so might consider just having one for simplicity). If the user is logged in, then show a profile icon to redirect to the profile.

Reply:
Yeah, I switched the navbar link on purpose for testing. Do we have a working login system yet? If so, I can look at implementing the login check and fix the navbar link.